hkm/static/hkm/js/hkm-v2.js

```diff
@@ -337,7 +343,50 @@ palikka
       });
     }
   }
+})
+.define('app.reset-password', ['jQuery', 'docReady'], function() {
+  $('#password-reset-email').on('submit', function(event) {
+    event.preventDefault();
 
+    $.post('/', {
+      action: 'password_reset',
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      email: $(this).find('input[name="email"]').val(),
+    }).done(function () {
+        $('#password-reset-form').addClass('hidden');
+        $('#password-reset-response-success').removeClass('hidden');
+    }).fail(function () {
+      $('#password-reset-form').addClass('hidden');
+      $('#password-reset-response-error').removeClass('hidden');
+    })
+  })
+
+  // Logic for opening the password reset modal automatically
+  const url = window.location.pathname;
```

Comment:
Strictly speaking you are accessing the pathname here. The URL would contain the protocol and so on.

Reply:
Changed this variable to `pathname`.